keel-web/config/keel.yml

```diff
@@ -6,6 +6,8 @@ keel:
     minAgeDuration: 10s
   artifact-refresh:
     frequency: PT1M
+  metadata-backfill:
```

Comment:
```suggestion
  artifact-metadata-backfill:
```

Reply:
Rather than having two configuration options with `artifact-` prefixes, I suggest doing `artifact.` instead, that way you could do them as subsections in the yaml if desired.